Sources/XCTest/Public/XCTestMain.swift

```diff
@@ -145,6 +139,15 @@ public func XCTMain(
         rootTestSuite.run()
         observationCenter.testBundleDidFinish(testBundle)
 
-        exit(rootTestSuite.testRun!.totalFailureCount == 0 ? EXIT_SUCCESS : EXIT_FAILURE)
+        return rootTestSuite.testRun!.totalFailureCount == 0 ? EXIT_SUCCESS : EXIT_FAILURE
     }
 }
+
+// @available(*, deprecated, message: "Call the overload of XCTMain() that returns an exit code instead.")
+public func XCTMain(
+    _ testCases: [XCTestCaseEntry],
+    arguments: [String] = CommandLine.arguments,
+    observers: [XCTestObservation]? = nil
```

Comment:
Using default parameters now means that it _could_ be source breaking if anyone is taking references to the prior functions. Could we keep the old overloads intact and just shim everything through?

Reply:
Blergh. Yeah, okay.